Web/Controllers/PaymentController.cs

```diff
@@ -70,50 +71,54 @@ public async Task<IActionResult> CreatePayment()
             return Ok(result);
         }
 
-        [HttpGet("payments/callback")]
-        public IActionResult PaymentCallback([FromQuery] string paymentId)
+        [HttpGet("payments/create/outbound")]
+        public async Task<IActionResult> CreateOutboundPayment()
         {
             if (_environment.IsProduction())
             {
                 return NotFound();
             }
-            if (string.IsNullOrWhiteSpace(paymentId))
+            IImmutableList<Account> accounts = ImmutableList.Create<Account>();
+            try
             {
-                return View("PaymentCallback",
-                            new PaymentCallbackViewModel
-                            {
-                                IsError = true
-                            });
+                accounts = await _viiaService.GetUserAccounts(User);
+            }
+            catch (ViiaClientException e)
+            {
+                // TODO
```

Comment:
What is TODO?

Reply:
Handle exception,but it is old I just moved it a bit around